packages/appsync-modelgen-plugin/src/visitors/appsync-javascript-visitor.ts

```diff
@@ -32,7 +32,10 @@ export class AppSyncModelJavascriptVisitor<
   TRawConfig extends RawAppSyncModelJavaScriptConfig = RawAppSyncModelJavaScriptConfig,
   TPluginConfig extends ParsedAppSyncModelJavaScriptConfig = ParsedAppSyncModelJavaScriptConfig
 > extends AppSyncModelTypeScriptVisitor<TRawConfig, TPluginConfig> {
-  protected IMPORT_STATEMENTS = ['import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";'];
+  protected IMPORT_STATEMENTS = [
+    'import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";',
+    'import { ManagedIdentifier, OptionallyManagedIdentifier, CustomIdentifier } from "@aws-amplify/datastore";',
```

Comment:
For some reason I can't see my requested changes comment on GH anymore, but could these imports be combined into one? Have multiple import statements from the same module often causes linting errors.